sormas-e2e-tests/src/test/resources/features/sanity/web/Contacts.feature

```diff
@@ -8,4 +8,15 @@ Feature: Create contacts
       And I create a new contact
       Then I check the created data is correctly displayed on Edit Contact page
       Then I open Contact Person tab
-      And I check the created data is correctly displayed on Edit Contact Person page
\ No newline at end of file
+      And I check the created data is correctly displayed on Edit Contact Person page
+
+  Scenario: Delete created contact
+    Given I log in with the user
+    And I click on the Contacts button from navbar
+    And I click on the NEW CONTACT button
+    And I create a new contact
+    Then I check the created data is correctly displayed on Edit Contact page
+    When I click on the Contacts button from navbar
+    Then I open the last created contact
+    Then I delete the contact
+    And I check that the last created contact was deleted
```

Comment:
This is not the best approach, please search for the id and check the number of the results. By using the wait until not visible the last step takes 30 seconds.